build.gradle.kts

```diff
@@ -5,4 +5,6 @@ plugins {
     alias(libs.plugins.androidLibrary) apply false
     alias(libs.plugins.jetbrainsCompose) apply false
     alias(libs.plugins.kotlinMultiplatform) apply false
+    alias(libs.plugins.sqlDelight).apply(false)
```

Comment:
Not pretty sure about this...
According to (https://github.com/Android-Dev-Peru/ADPMeetUps/blob/dev/docs/design-docs/01%20-%20mvp/01-mvp-design.md) →  "_Disclaimer: implementation details for external data sources or backend -> TBD!_"

I though we were going to discuss if we are going to use Firebase first, or another type of external data source 🤔

Reply:
add offline mode, as part of the documentation